boa/contracts/vyper/contract.py

```diff
@@ -22,34 +22,42 @@
 from vyper.codegen.global_context import GlobalContext
 from vyper.codegen.ir_node import IRnode
 from vyper.codegen.module import generate_ir_for_module
+from vyper.compiler import CompilerData
 from vyper.compiler import output as compiler_output
 from vyper.compiler.settings import OptimizationLevel
 from vyper.evm.opcodes import anchor_evm_version
 from vyper.exceptions import VyperException
 from vyper.ir.optimizer import optimize
 from vyper.semantics.analysis.data_positions import set_data_positions
-from vyper.semantics.types import AddressT, EventT, HashMapT, TupleT
-from vyper.semantics.types.function import ContractFunctionT
+from vyper.semantics.types import AddressT, HashMapT, TupleT
 from vyper.utils import method_id
 
+from boa.contracts.evm_contract import BaseEVMContract
+from boa.contracts.vyper import _METHOD_ID_VAR
```

Comment:
bit weird to be importing from the parent folder -- let's move `_METHOD_ID_VAR` to compiler_utils